src/utils/partials/alert.hbs

```diff
@@ -0,0 +1,23 @@
+<head>
+  <link rel="stylesheet" href="/technical-styles/alert.css" />
+  <script src="/technical-scripts/alert.js" defer></script>
+</head>
+
+<body>
+  <section>
+    <div
+      class="modal fade custom-alert"
```

Comment:
Format this code block

Reply:
This is to avoid maximum length of a line